R/stacks.R

```diff
@@ -0,0 +1,48 @@
+#' @rdname vetiver_create_description
+#' @export
+vetiver_create_description.model_stack <- function(model) {
+    num_members <-
+        tidy(model$coefs) %>%
+        dplyr::filter(estimate > 0, term != "(Intercept)") %>%
+        nrow()
+    glue("A {model$mode} stacked ensemble with {num_members} members")
+}
+
+#' @rdname vetiver_create_meta
+#' @export
+vetiver_create_meta.model_stack <- function(model, metadata) {
+    reqs <- map(model$member_fits, required_pkgs)
+    reqs <- purrr::flatten_chr(reqs)
+    reqs <- sort(unique(c(reqs, "workflows", "stacks")))
```

Comment:
The `model$coefs` slot has a glmnet dependency, and may introduce others if we move forward with generalizing the meta-learner. Maybe:

```suggestion
    reqs <- sort(unique(c(reqs, required_pkgs(model$coefs), "workflows", "stacks")))
```